primer/index.bs

```diff
@@ -535,60 +521,61 @@ Our JWK thumbprint looks more like:
 Since `openid` was listed as a scope during the authorization request, the OP generates an id
 token. The id token will be pushed as claim to any Solid Authorization Server needed, which clients want to get Access Token from.
 This token is a [JSON Web Token](https://jwt.io/introduction/). It would look like
-the following (you can decrypt the token with [https://jwt.io](https://jwt.io/#debugger-io?token=eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJhdWQiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL3dlYmlkI3RoaXMiLCJ3ZWJpZCI6Imh0dHBzOi8vYWxpY2UuY29vbHBvZC5leGFtcGxlL3Byb2ZpbGUvY2FyZCNtZSIsImlzcyI6Imh0dHBzOi8vc2VjdXJlYXV0aC5leGFtcGxlIiwianRpIjoiODQ0YTA5NWMtOWNkYi00N2U1LTk1MTAtMWRiYTk4N2MwYTVmIiwiaWF0IjoxNjAzMzg2NDQ4LCJleHAiOjE2MDMzODcwNDh9.T306vT8dmn9gQIMEdG92AM4WRnrhqWZTfDpovwqZ6Zn0mK9yxj0iOVGqXD4CW8-tzDTitNwEGorAo85atL0Oeg)):
-
-
-Issue: encoded token needs to be updated
+the following (you can decrypt the token with [https://jwt.io](https://jwt.io/#debugger-io?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJhdWQiOlsic29saWQiLCJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL2NsaWVudF9pZCJdLCJhenAiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL2NsaWVudF9pZCIsIndlYmlkIjoiaHR0cHM6Ly9hbGljZS5jb29scG9kLmV4YW1wbGUvcHJvZmlsZS9jYXJkI21lIiwiaXNzIjoiaHR0cHM6Ly9zZWN1cmVhdXRoLmV4YW1wbGUiLCJqdGkiOiI4NDRhMDk1Yy05Y2RiLTQ3ZTUtOTUxMC0xZGJhOTg3YzBhNWYiLCJpYXQiOjE2MDMzODY0NDgsImV4cCI6MTYwMzM4NzA0OCwiY25mIjp7ImprdCI6IjlYbXdLOG1RM0g1LVBuekF0M2xGSHpXQldfdjVRaFl5bmV6YmJpdDRrQzgifX0.zArlHB_4ESTQRXQEzTWS5e7QhnrjuBd6vocnNj1XzOg):
 
 ```text
-eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJhdWQiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL3dlYmlkI3RoaXMiLCJ3ZWJpZCI6Imh0dHBzOi8vYWxpY2UuY29vbHBvZC5leGFtcGxlL3Byb2ZpbGUvY2FyZCNtZSIsImlzcyI6Imh0dHBzOi8vc2VjdXJlYXV0aC5leGFtcGxlIiwianRpIjoiODQ0YTA5NWMtOWNkYi00N2U1LTk1MTAtMWRiYTk4N2MwYTVmIiwiaWF0IjoxNjAzMzg2NDQ4LCJleHAiOjE2MDMzODcwNDh9.T306vT8dmn9gQIMEdG92AM4WRnrhqWZTfDpovwqZ6Zn0mK9yxj0iOVGqXD4CW8-tzDTitNwEGorAo85atL0Oeg
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJodHRwczovL2FsaWNlLmNvb2xwb2QuZXhhbXBsZS9wcm9maWxlL2NhcmQjbWUiLCJhdWQiOlsic29saWQiLCJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL2NsaWVudF9pZCJdLCJhenAiOiJodHRwczovL2RlY2VudHBob3Rvcy5leGFtcGxlL2NsaWVudF9pZCIsIndlYmlkIjoiaHR0cHM6Ly9hbGljZS5jb29scG9kLmV4YW1wbGUvcHJvZmlsZS9jYXJkI21lIiwiaXNzIjoiaHR0cHM6Ly9zZWN1cmVhdXRoLmV4YW1wbGUiLCJqdGkiOiI4NDRhMDk1Yy05Y2RiLTQ3ZTUtOTUxMC0xZGJhOTg3YzBhNWYiLCJpYXQiOjE2MDMzODY0NDgsImV4cCI6MTYwMzM4NzA0OCwiY25mIjp7ImprdCI6IjlYbXdLOG1RM0g1LVBuekF0M2xGSHpXQldfdjVRaFl5bmV6YmJpdDRrQzgifX0.zArlHB_4ESTQRXQEzTWS5e7QhnrjuBd6vocnNj1XzOg
 ```
 
 Token Header:
+
 ```json
 {
-    "alg": "ES256",
+    "alg": "HS256",
```

Comment:
No,we should not use HS256 in examples

Reply:
In which case, we need to include the example public/private key such that we can regenerate the values in the document.

Without these, we'll continue to get issues with the code samples not being entirely correct.